prime-router/settings/staging/0026-CA.yml

```diff
@@ -0,0 +1,49 @@
+# Goal: Turn off the procesing_mode_code test for CA in staging, so data can flow to at least one state.
+# Change:  The qualityFilter is changed.  Same as default, minus the processing_mode_code test.
+---
+name: "elr"
+organizationName: "ca-dph"
+topic: "covid-19"
+translation: !<HL7>
+  useTestProcessingMode: false
+  useBatchHeaders: true
+  receivingApplicationName: null
+  receivingApplicationOID: null
+  receivingFacilityName: null
+  receivingFacilityOID: null
+  messageProfileId: null
+  reportingFacilityName: null
+  reportingFacilityId: null
+  suppressQstForAoe: false
+  suppressHl7Fields: null
+  suppressAoe: false
+  defaultAoeToUnknown: false
+  useBlankInsteadOfUnknown: null
+  truncateHDNamespaceIds: false
+  usePid14ForPatientEmail: false
+  convertTimestampToDateTime: null
+  processingModeCode: null
+  nameFormat: "STANDARD"
+  receivingOrganization: null
+  type: "HL7"
+jurisdictionalFilter:
+- "orEquals(ordering_facility_state, CA, patient_state, CA)"
+qualityFilter:
+- "hasValidDataFor(message_id,equipment_model_name,specimen_type,test_result,patient_last_name,patient_first_name,patient_dob)"
+- "hasAtLeastOneOf(patient_street,patient_zip_code,patient_phone_number,patient_email)"
+- "hasAtLeastOneOf(order_test_date,specimen_collection_date_time,test_result_date)"
+- "isValidCLIA(testing_lab_clia,reporting_facility_clia)"
+reverseTheQualityFilter: false
```

Comment:
I changes CA (only) in Staging to NOT use the default Quality filter.  Instead it has the entire filter EXCEPT the test that processing_mode_code must be 'P'.   This is to allow testing data (in mode 'T') to flow on to California, for testing.